doxygen: fix empty documentation

rjarry opened the pull request:
The headers have been moved/renamed. The generated documentation is empty.

Fix Doxyfile.in.

rjarry committed:
doc: fix doxygen configuration

The headers have been moved to a subdir and renamed. Adapt Doxyfile.in
accordingly.

Fixes: 87fe941e9340 ("include: move sub-headers to sub-directory")
Signed-off-by: Robin Jarry <robin@jarry.cc>
Reviewed-by: Olivier Matz <zer0@droids-corp.org>

rjarry committed:
editline: add missing doxygen group delimiters

Add opening and closing bracket for the ecoli_editline group.

Fixes: 38e3daa01a2b ("editline: add api documentation")
Signed-off-by: Robin Jarry <robin@jarry.cc>
Reviewed-by: Olivier Matz <zer0@droids-corp.org>

zer0-droids commented on include/ecoli/editline.h:
```diff
@@ -4,6 +4,7 @@
 
 /**
  * @defgroup ecoli_editline Editline
+ * @{
```
The same thing (almost) is missing un ec_complete api ? Can you add it in your PR ? I have it locally, so if you prefer I can submit it.

rjarry replied:
done

rjarry committed:
complete: move doxygen group terminator

The group is opened and closed before symbols are defined. Close it at
the end of the file.

Fixes: e18710da81b4 ("add first documentation draft and framework")
Signed-off-by: Robin Jarry <robin@jarry.cc>
Reviewed-by: Olivier Matz <zer0@droids-corp.org>

zer0-droids commented on include/ecoli/complete.h:
```diff
@@ -508,3 +508,5 @@ struct ec_comp_item *ec_comp_iter_next(struct ec_comp_item *item, enum ec_comp_t
 #define EC_COMP_FOREACH(item, comp, type)                                                          \
 	for (item = ec_comp_iter_first(comp, type); item != NULL;                                  \
 	     item = ec_comp_iter_next(item, type))
+
```
There is a "@}" at the beginning of the file too. I think it should be removed.

rjarry replied:
Nice catch. Fixed.
